### PR TITLE
[stable/prometheus-operator] deprecate chart

### PR DESCRIPTION
**Pull request by scottrigby**
#### What this PR does / why we need it:

This chart has successfully moved 🎉

#### Special notes for your reviewer:

- Completed in https://github.com/prometheus-community/helm-charts/pull/1
- Related to https://github.com/helm/charts/issues/21103

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
~- [ ] Variables are documented in the README.md~
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/prometheus-operator/Chart.yaml [hunk 1]
```diff
@@ -1,18 +1,16 @@
 apiVersion: v1
-description: Provides easy monitoring definitions for Kubernetes services, and deployment and management of Prometheus instances.
+# The foo chart is deprecated and no longer maintained. For details deprecation,
```
Comment (unguiculus) [inline stable/prometheus-operator/Chart.yaml]:
```suggestion
# The prometheus-operator chart is deprecated and no longer maintained. For details deprecation,
```

**Review by unguiculus — DISMISSED** [review]

File: stable/prometheus-operator/Chart.yaml (hunk 1 above)
Comment (scottrigby) [reply stable/prometheus-operator/Chart.yaml]: Thanks lol

**Comment by scottrigby** [comment]
Looks like test infra had issues earlier, and this has been stuck for a few hours

**Comment by scottrigby** [comment]
Not sure we even need to test this, as it's just a deprecation. But let's see if it does it on it's own now.

**Comment by scottrigby** [comment]
It's stuck again. Merging

**Pull request merged**